apisix/cli/ngx_tpl.lua

```diff
@@ -765,6 +765,18 @@ http {
             }
             {% end %}
 
+
+            {% if proxy_mirror_timeouts then %}
+                {% if proxy_mirror_timeouts.connect then %}
+            proxy_connect_timeout {* proxy_mirror_timeouts.connect *}ms;
```

Comment:
Would be better to put the time unit in the configuration value instead of hardcoded it.

Reply:
How about keep the same to the nginx configuration and let users choose seconds or milliseconds by themselves